### ISSUE TITLE
Plug-in to create oral version of MAST Session 2 of Day 1 Activity

### ISSUE THREAD

**Issue by Redmayspecial** [WycliffeAssociates/5WoC#1013]
As a facilitator, I would like plug support in tR so that I can have translators and checkers perform Session 2 of Day 1 Activities in the MAST training.

Dan Kramer
   |
Today, 7:01 AM
Hi Bob 
We just had the same situation in China.  We used any random chapter and chunks for the oral recording of the steps.   You can compile it (like a chapter) if you wanted to have it in a recording to take but we were more worried about the exercise then the end product. 

Not sure if that's all you're asking but if not, please keep asking :)
Dan

Sent from my iPhone
 
Bob Chambers

   Reply all|
Today, 6:57 AM
Dan Kramer;
Sam Todd;
Glenn Oliver 
We are working through the Day 1 material. Our first OL work is later this month and it will be an oral project. How do we work with the activity sentence ("Hi, my name is Sam ...") and the 8 MAST steps. Is there a place in tR to create an oral version of the activity sentence so we can make that the source text for Day 1 activity?

In other words, since there's no orthography for this event's target language, how do we take them through Session 1 of Day 1? The translators will be writing the activity sentence using steps 1 - 5 and then editing using steps 6-8. But there's no writing at this event. And they will need an oral version of the steps for Session 2 of Day 1.

Bob
